snmp/datadog_checks/snmp/data/default_profiles/chatsworth_pdu.yaml

```diff
@@ -483,38 +1484,44 @@ metrics:
 
   ### Locks
 
-  - # This table contains a list of electronic locks that are connected to the PDU
-    MIB: CPI-UNITY-MIB
+    # This table contains a list of electronic locks that are connected to the PDU
+  - MIB: CPI-UNITY-MIB
     table:
       OID: 1.3.6.1.4.1.30932.1.10.1.7.100
-      name: cpiPduOutletTable
+      name: cpiEasTable
     symbols:
         # Indicates if the Electronic Access Control system is ready, inactive, or in an error state: inactive(0), ready(1), error(2)
       - OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.3
         name: cpiPduEasStatus
-      - # Indicates if the door sensor is closed or open: closed(0), open(1)
-        OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.4
+        # Indicates if the door sensor is closed or open: closed(0), open(1)
+      - OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.4
         name: cpiPduDoorStatus
-      - # Indicates if the lock is closed or open: closed(0), open(1)
-        OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.5
+        # Indicates if the lock is closed or open: closed(0), open(1)
+      - OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.5
         name: cpiPduLockStatus
     metric_tags:
       - tag: lock_id
         column:
           OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.2
           name: cpiPduLockId
+        mapping:
+          1: front
+          2: rear
   - MIB: CPI-UNITY-MIB
     table:
       OID: 1.3.6.1.4.1.30932.1.10.1.7.100
-      name: cpiPduOutletTable
```

Comment:
The fix should also contain cpiPduOutletTable -> cpiEasTable?

Reply:
Fixed in f87c27b